packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -48,13 +48,14 @@ class Layout extends React.Component {
             {intl.formatMessage({ id: 'Terra.Consumer.Layout.skipcontent' })}
           </a>
         </div>
-        <div className={cx('layout', customProps.className)} {...customProps}>
-          <Nav
-            {...nav}
-            isMobileNavOpen={this.state.isMobileNavOpen}
-            onRequestClose={this.toggleNav}
-          />
-          <main id="main-container" className={cx('main-container', this.state.isMobileNavOpen && 'nav-open')}>
+        <div className={cx('layout', { open: this.state.isMobileNavOpen }, customProps.className)} {...customProps}>
```

Comment:
your customProps spat needs to go first. If className is supplied it will override the component defined className I think